clang/test/Driver/sse2avx.s

```diff
@@ -0,0 +1,89 @@
+# RUN: %clang -march=x86-64 -msse2avx %s -c -o %t
```

Comment:
Do not add end2end test. Check test for other flags.

Reply:
Done